tests/local/network.go

```diff
@@ -464,7 +471,7 @@ func (n *LocalNetwork) AddSubnetValidators(ctx context.Context, subnetID ids.ID,
 
 	// consume some of the extraNodes
 	newValidatorNodes := n.extraNodes[0:count]
-	defer slices.Delete(n.extraNodes, 0, int(count))
+	copy(n.extraNodes, n.extraNodes[count:])
```

Comment:
This will still preserve the nodes to be deleted in `n.extraNodes`, since `n.extraNodes[count:]` is a subset. See example here: https://go.dev/play/p/z8RhPp136sr

Reply:
Fixed by copying the `newValidatorNodes` from `extraNodes` and then just assigning `extraNodes` to the subset.